open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/package.json

```diff
@@ -29,19 +30,19 @@
     "@polymer/iron-selector": "^3.0.1",
     "@polymer/neon-animation": "^3.0.1",
     "@polymer/paper-button": "^3.0.1",
+    "@polymer/paper-dialog": "^3.0.1",
```

Comment:
Paper dialog is not friendly with mobile version. Or difficult to customize for mobile mersion.
For POC is fine.